Language/Classes/Instance_Methods/Method_noSuchMethod/superclass_A01_t02.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion It is a compile-time error if the name m is noSuchMethod
+/// forwarded in a concrete class C, and a superclass of C has an accessible
+/// concrete declaration of m which is not a noSuchMethod forwarder.
+///
+/// @description Checks that it is a compile-time error if the name `m` is
+/// `noSuchMethod` forwarded in a concrete class `C`, and a superclass of `C`
+/// has an accessible concrete declaration of `m` which is not a `noSuchMethod`
+/// forwarder.
```

Comment:
An extra sentence would probably be helpful here, too, similarly to Language/Classes/Instance_Methods/Method_noSuchMethod/superclass_A01_t01.dart.